Zappy_IA_Report/DTC/Video/Tutorial4.py

```
@@ -0,0 +1,32 @@
+import pandas as pd
+
+df = pd.read_csv('500hits.csv', encoding='latin-1')
+print(df.head())
+
+df = df.drop(columns= ['PLAYER', 'CS'])
+X = df.iloc[:, 0:13]
+y = df.iloc[:, 13]
+
+from sklear.model_selection import train_test_split
```

Ruff (0.11.9)

Module level import not at top of file

(E402)